### ISSUE TITLE
Using isar outside of dart?

### ISSUE THREAD

**Issue by morr0ne** [isar/isar#8]
I wanted to open this on [isar-core](https://github.com/isar/isar-core) but issues seem to be disabled
Being that isar is written in rust, a relatively low-level system programming language, would it be possible to be used in another context? E.g. rust application, binding to other languages, or has it been strictly written for use on mobile and web apps?

**Comment by leisim** [comment]
Yes, I definitely plan to allow standalone usage of isar core. Currently it is not ready yet however. It will work very well on mobile phones, desktops etc. In theory, it should also be a good fit for servers but I need to do a lot of testing to confirm that.

**Comment by morr0ne** [comment]
Is there a particular reason issues can't be open on isar-core?

**Comment by leisim** [comment]
No I'll change that.

**Comment by morr0ne** [comment]
Would it make sense to move this issue on isar-core in case further discussion is needed in the future?

**Comment by leisim** [comment]
Yes I'll do that but I already planned supporting isar core as standalone
